appcelerator/titanium_mobile
# fix(android): allow requestLegacyExternalStorage attribute

- Add `requestLegacyExternalStorage` to attributes

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-27381)

**Test:**
1. Create a Classic Titanium project.
2. Add the below `<application/>` element to the project's "tiapp.xml" file.
3. Build for Android.
4. In Finder (or Windows Explorer) go to project folder: `./build/android`
5. Open the "AndroidManifest.xml" file.
6. Verify that the `<application/>` element has attribute `android:requestLegacyExternalStorage="true"`.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<ti:app xmlns:ti="http://ti.appcelerator.org">
	<android xmlns:android="http://schemas.android.com/apk/res/android">
		<manifest>
			<application android:requestLegacyExternalStorage="true"/>
		</manifest>
	</android>
</ti:app>
```
